### PR TITLE
Add functional test for change email

### PR DESCRIPTION
**Pull request by kentsanggds**
## What

Added functional test for change email

## How to review

execute run-tests

### CODE REVIEW

File: tests/functional/conftest.py [hunk 1]
```diff
@@ -37,3 +37,10 @@ def do_submit_known_email_address(email_address):
         set_email_address(email_address)
         browser.find_by_css('form button').click()
     return do_submit_known_email_address
+
+
+@pytest.fixture
+def click_change_email_link_on_dept_confirm(browser):
```
Comment (andyhd) [inline tests/functional/conftest.py]: This function name is too long. Ideally, names should contain fewer than 4 words. How about `click_change_email`?

File: tests/functional/conftest.py [hunk 2]
```diff
@@ -37,3 +37,10 @@ def do_submit_known_email_address(email_address):
...
+    def do_click_change_email_link_on_dept_confirm():
+        browser.find_by_css('#confirm-dept a').click()
```
Comment (andyhd) [inline tests/functional/conftest.py]: This CSS selector isn't very specific - maybe add an `id` attribute to the link in the template and use that in the selector

File: tests/functional/test_confirm_department.py [hunk 3]
```diff
@@ -1,28 +1,31 @@
...
 
-    def it_goes_to_confirm_email_when_continue_clicked(
-            self, live_server, browser):
-        browser.visit(url_for('main.confirm_dept', _external=True))
+    @pytest.fixture(autouse=True)
+    def setup_page(self, email_address, submit_known_email_address):
+        submit_known_email_address(email_address)
 
+    def it_goes_to_confirm_email_when_continue_clicked(self, browser):
         browser.choose('confirm', 'no')
```
Comment (andyhd) [inline tests/functional/test_confirm_department.py]: This should go in the `setup_page` method

**Review by andyhd — CHANGES_REQUESTED** [review]

File: tests/functional/conftest.py (hunk 1 above)
Comment (kentsanggds) [reply tests/functional/conftest.py]: ok

File: tests/functional/test_confirm_department.py (hunk 3 above)
Comment (kentsanggds) [reply tests/functional/test_confirm_department.py]: true

File: tests/functional/conftest.py (hunk 2 above)
Comment (kentsanggds) [reply tests/functional/conftest.py]: had thought about whether to add an id as it is self contained, but an id would prevent confusion if another link was added in same containing element

File: tests/functional/conftest.py [hunk 4]
```diff
@@ -40,7 +40,7 @@ def do_submit_known_email_address(email_address):
...
 
 @pytest.fixture
-def click_change_email_link_on_dept_confirm(browser):
-    def do_click_change_email_link_on_dept_confirm():
-        browser.find_by_css('#confirm-dept a').click()
-    return do_click_change_email_link_on_dept_confirm
+def click_change_email(browser):
+    def do_click_change_email():
+        browser.find_by_css('#change_email').click()
+    return do_click_change_email
```
Comment (andyhd) [inline tests/functional/conftest.py]: This one might be unnecessary now - `browser.find_by_css('#change_email').click()` is pretty self-explanatory.
Just pass `browser` into the `setup_page` method and execute this directly.

**Review by andyhd — CHANGES_REQUESTED** [review]

File: tests/functional/conftest.py (hunk 4 above)
Comment (kentsanggds) [reply tests/functional/conftest.py]: ok

**Review by andyhd — APPROVED** [review]

**Pull request merged**